Libplanet/Action/ActionEvaluator.cs

```diff
@@ -483,9 +483,11 @@ ActionContext CreateActionContext(IAccountStateDelta prevStates, int randomSeed)
                     .ForContext("Tag", "Metric")
                     .ForContext("Subtag", "TxEvaluationDuration")
                     .Debug(
-                        "{ActionCount} actions in transaction {TxId} by {Signer} " +
+                        "{ActionCount} actions {ActionTypes} in transaction {TxId} by {Signer} " +
                         "with timestamp {TxTimestamp} evaluated in {DurationMs:F0}ms.",
                         tx.Actions.Count,
+                        tx.Actions.Select(action => action.ToString()!.Split('.')
+                            .LastOrDefault()?.Replace(">", string.Empty)),
```

Comment:
https://github.com/planetarium/libplanet/pull/1909#discussion_r848292179

I think you might've missed this comment. 😥

```suggestion
                        tx.Actions.Select(action => action.GetType().Name),
```

I think this should also work.

Reply:
I'd like to vote for @greymistcube's way, because not every `IAction.ToString()` implementation shows its type name.  However it seems still half-baked too, as some *higher-order* actions like `PolymorphicAction<T>` would hide the type of its actual action.  🤔

Reply:
We may have to override `ToString()` in `PolymorphicAction<T>` type... I think that it's possible option because [current `PolymorphicAction<T>.ToString()` isn't enough](https://github.com/planetarium/libplanet/pull/1909#discussion_r848329829) atm.

Reply:
See https://github.com/planetarium/libplanet/pull/1909#issuecomment-1096675500 below for a bit more baked version. 😗

Reply:
@longfin We should replace `_innerAction` with `_innerAction.GetType().Name` here:

https://github.com/planetarium/libplanet/blob/1398c43139ad860e373f6fbecbeda06c2e2ddfd3/Libplanet/Action/PolymorphicAction.cs#L253-L259

Reply:
Doesn't this only reduce the clutter for the sake of it?
That would change

```
Libplanet.Action.PolymorphicAction<Nekoyume.Action.HackAndSlash>
```

to

```
Libplanet.Action.PolymorphicAction<HackAndSlash>
```

which could be confusing if we don't use its fully qualified name. 🙄

Reply:
@greymistcube Hmm… you're right.  @longfin Apparently `PolymorphicAction<T>.ToString()` already gives enough information.  It's probably JetBrains Rider's own behavior.  Or maybe we need to customize [debug display][1]?

[1]: https://docs.microsoft.com/en-us/dotnet/framework/debug-trace-profile/enhancing-debugging-with-the-debugger-display-attributes

Reply:
Also considering its recursive usage of generic type parameter, from `PolymorphicAction<T> : T`, I think it is more expected to call `T.ToString()` on `T`. 😶